src/main/java/org/mule/tools/maven/plugin/module/analyze/AnalyzeMojo.java

```diff
@@ -121,9 +123,9 @@ private boolean checkDependencies() throws MojoExecutionException {
       throw new MojoExecutionException("Cannot analyze module API", exception);
     }
 
-    final HashMap<String, Set<String>> undeclaredExportedPackages =
-        new HashMap<String, Set<String>>(analysis.getUndeclaredPackageDeps());
-    final HashSet<String> packagesToExport = new HashSet<String>(analysis.getPackagesToExport());
+    final Map<String, Set<String>> undeclaredExportedPackages = new HashMap<>(analysis.getUndeclaredPackageDeps());
+    final Set<String> packagesToExport = new HashSet<>(analysis.getPackagesToExport());
+    final Set<String> noAnalyzedPackages = new HashSet<>(analysis.getNoAnalyzedPackages());
```

Comment:
rename attribute and method to `notAnalyzedPackages` and `getNotAnalyzedPackages()`